docs/README.md

```diff
@@ -0,0 +1,2 @@
+# Docs
+This directory will be used to generate docs of this SDK based on the Python code.
```

Comment:
We don't typically store any kind of docs in our repo like this.  Is this a Python thing? If you are wanting to do auto-generated docs like `javadoc`, then there is a different way to handle it.

Reply:
I was aiming for the auto-generated docs, yes. How would you suggest going about it?